lib/connectasmediaextension.js

```diff
@@ -0,0 +1,23 @@
+/* globals context */
+'use strict';
+
+const CancelablePromise = require('./util/cancelablepromise');
+const { typeErrors: E } = require('./util/constants');
+
+/**
+ * Connect as a media extension {@link Participant}.
+ * @param {ConnectOptions} [options]
+ * @returns {CancelablePromise<Room>}
+ */
+function connectAsMediaExtension(options = {}) {
+  if (typeof context === 'object' && typeof context.connectAsMediaExtension === 'function') {
```

Comment:
Is this the whole implementation of this video sdk api? What is the benefit of adding this to the sdk if the application already have access to `context.connectAsMediaExtension`?

Reply:
We discussed about who should own this function, and it was agreed that the SDK will own it, because we need some place where we can provide documentation and examples. The internal context-based function will be managed by the back-end team, but it will not be public.